client/pages/index.jsx

```diff
@@ -1,13 +1,121 @@
-import React from "react";
-import { Head } from "../components";
-import { Button } from "reactstrap";
+import React, { Component } from "react";
+import { Head } from "../components/Head";
+import ReactPlayer from "react-player";
+import {
+  Button,
+  Card,
+  CardBody,
+  CardGroup,
+  CardImg,
+  Col,
+  Container,
+  Input,
+  Row
+} from "reactstrap";
 
-export default function() {
-  return (
-    <>
-      <Head title="Home" />
-      <p>Hello Next.js</p>
-      <Button>Hello</Button>
-    </>
-  );
+import "../public/style.scss";
+
+export default class App extends Component {
+  sdgImages = [];
+  populateImagesArr = sdgImages => {
+    for (let i = 1; i <= 17; i++) {
+      const sdgObject = {
+        id: i,
+        imageLink: `/sdg-images/${i}.png`
+      };
+      sdgImages.push(sdgObject);
+    }
+    const sdgObject = {
+      id: 18,
+      imageLink: `/sdg-images/sdg.png`
+    };
+    sdgImages.push(sdgObject);
+  };
+
+  render() {
+    this.populateImagesArr(this.sdgImages);
+    return (
+      <Container>
+        <Row className="header-row" justify="center" align="middle">
+          <Col xs="1">
+            <img className="header-img" src="/homepage-images/menu-icon.png" />
+          </Col>
+          <Col xs="1">
+            <img className="header-img" src="/homepage-images/kso-icon.png" />
+          </Col>
+          <Col xs={{ size: 1, offset: 8 }}>
+            <img
+              className="header-img"
+              src="/homepage-images/notification-icon.png"
+            />
+          </Col>
+          <Col xs="1">
+            <img className="header-img" src="/homepage-images/usericon.png" />
+          </Col>
+        </Row>
+        <div className="page-title">
+          <h1 className="header2-text" align="center">
+            <strong>FateMaker</strong>
+          </h1>
+        </div>
+        <Row className="home-block-1" type="flex" justify="center" xs="2">
+          <Col className="home-block-col">
+            <ReactPlayer url="https://youtu.be/dQw4w9WgXcQ" />
+          </Col>
+          <Col className="home-block-col">
+            <a className="text-link" href="about us">
+              ABOUT US
+            </a>
+            <h1 className="header-text">Change the World,</h1>
+            <h1 className="header-text" mode="single">
+              one step at a time.
+            </h1>
+            <p className="body-text">
+              The barred ratio flips under the proved nail. An umbrella brushes
```

Comment:
where is this from lol

Reply:
It's filler text it was a random paragraph generator